images/builders/mlflow/README.md

```diff
@@ -0,0 +1,146 @@
+# MLflow python environment builder component for FuseML workflows
+
+## Overview
+
+The [MLFlow Project format](https://www.mlflow.org/docs/latest/projects.html) is a flexible way of configuring how to package and execute python ML code in a reproducible way. It allows developers to declaratively specify python package requirements and code execution entry points.
+
+The Dockerfile and associated files in this folder implement the FuseML MLflow builder workflow step. The container image is based on [Kaniko](https://github.com/GoogleContainerTools/kaniko) and leverages the MLflow Project conventions to automate building MLflow runtime environments: container images used for the execution of MLflow augmented python code within a FuseML workflow.
+
+The MLflow builder works with any codeset that meets the following requirements and conventions:
+
+- an `MLproject` file is present the codeset's root directory
+- python requirements are specified using a `conda.yaml` file or a `requirements.txt` file, also present in the codeset's root directory
+- the `MLproject` file describes one or more entry points for the codeset's python code, as well as a list of parameters that are passed to the entry points. The entry points are specified using the `entry_points` key in the `MLproject` file. At a minimum, a `main` entrypoint is required, which will be used as the default entry point when executing the codeset code, unless overridden by workflow input parameters. An example `MLproject` file and a corresponding `conda.yaml` file are shown below:
+
+    ```yaml
+    name: my_keras_model
+    conda_env: conda.yaml
+    entry_points:
+      main:
+        parameters:
+          epochs: {type: int, default: 2}
+          batch_size: {type: int, default: 64}
+        command: "python train.py --epochs={epochs} --batch_size={batch_size}"
+    ```
+
+    ```yaml
+    name: my_test_env
+    channels:
+      - conda-forge
+    dependencies:
+      - python=3.6
+      - pip
+      - pip:
+        - mlflow
+        - tensorflow==2.0.0
+    ```
+
+The MLflow builder has a single output: the container registry repository and tag where the built MLflow environment container image is stored. This output can be used in subsequent workflow steps to run the MLflow code from the same codeset as the one used as input.
```

Comment:
_a/the_ tag I assume ... although that sentence reads quite strange: "the registry and the tag where ... image is stored"

Reply:
corrected